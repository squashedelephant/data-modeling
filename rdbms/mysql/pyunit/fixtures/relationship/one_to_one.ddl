CREATE TABLE color(id INT NOT NULL AUTO_INCREMENT, name VARCHAR(20), PRIMARY KEY (id));
CREATE TABLE balloon(id INT NOT NULL, name CHAR(15) NOT NULL, total INT NOT NULL, PRIMARY KEY (id));
