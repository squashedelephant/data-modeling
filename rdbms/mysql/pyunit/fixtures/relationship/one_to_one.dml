INSERT INTO color(name) VALUES('red');
SET @color_id = LAST_INSERT_ID();
INSERT INTO balloon (id, name, total) VALUES (@color_id, 'holiday', 3);
INSERT INTO color(name) VALUES('pink');
SET @color_id = LAST_INSERT_ID();
INSERT INTO balloon (id, name, total) VALUES (@color_id, 'birthday', 1);
INSERT INTO color(name) VALUES('blue');
SET @color_id = LAST_INSERT_ID();
INSERT INTO balloon (id, name, total) VALUES (@color_id, 'friendship', 5);
