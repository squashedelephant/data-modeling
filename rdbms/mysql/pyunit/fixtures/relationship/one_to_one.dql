SELECT b.name, c.name, b.total FROM balloon AS b INNER JOIN color AS c ON b.id = c.id;
