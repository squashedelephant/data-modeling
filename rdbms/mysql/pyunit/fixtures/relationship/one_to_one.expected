holiday
red
3L
birthday
pink
1L
friendship
blue
5L
